src/libs/SidebarUtils.ts

```diff
@@ -654,6 +654,7 @@ function getOptionData({
     reportAttributes: OnyxEntry<ReportAttributes>;
     card: Card | undefined;
     lastAction: ReportAction | undefined;
+    translate: LocaleContextProps['translate'];
```

Comment:
```suggestion
    translate: LocalizedTranslate;
```

Reply:
@parasharrajat Is there a reason for this change? Both meant the same thing.